notebooks/use extractor.ipynb

```diff
@@ -1,111 +0,0 @@
-{
```

Comment:
Removed as it seems out of date, with references to the missing `FormExtractor` class.